src/effects/specialeffectchainslots.cpp

```diff
@@ -136,6 +145,16 @@ EqualizerEffectChainSlot::EqualizerEffectChainSlot(const QString& group,
     setupLegacyAliasesForGroup(group);
 }
 
+void EqualizerEffectChainSlot::loadEffect(
+        const unsigned int iEffectSlotNumber,
+        const EffectManifestPointer pManifest,
+        std::unique_ptr<EffectProcessor> pProcessor) {
+    // TODO: preserve effect parameters when loading new effect. This will allow
+    // for easy comparison of the sound of different equalizer effects.
```

Comment:
Let's take care of this TODO after removing EffectParameter. I think it should be implemented by passing an optional bool to EffectChainSlot::loadEffect and down to EffectSlot::loadEffect so EffectChainSlot does not need to access any private information about the state of EffectParameterSlots.